src/lexedata/change/clean_ids.py

```diff
@@ -53,6 +53,47 @@ def clean_mapping(rows: t.Mapping[str, t.Mapping[str, str]]) -> t.Mapping[str, s
     return mapping
 
 
+def update_integer_ids(ds: pycldf.Dataset, table: csvw.metadata.Table):
```

Comment:
@Anaphory I think `mapping: t.Mapping[str, str]` might be missing as a function argument or mapping.get shouldn't be here. When you call the function update_integer_ids() in the __main__ part, mapping is not yet defined.

Reply:
Probably true! I didn't think the integer IDs through very well, I just noticed a huge gap in the code.

Reply:
I fixed that step now. The functions are very similar, they could be generalized into one with two additional arguments, I think.